src/test/rgw/test_multi.py

```diff
@@ -23,12 +22,6 @@
 from rgw_multi.zone_az import AZone as AZone
 from rgw_multi.zone_az import AZoneConfig as AZoneConfig
 
-# make tests from rgw_multi.tests available to nose
-from rgw_multi.tests import *
-from rgw_multi.tests_es import *
-from rgw_multi.tests_ps import *
-from rgw_multi.tests_az import *
```

Comment:
i tried running this modified `test_multi.py` by hand (according to docs in https://github.com/ceph/ceph/blob/master/src/test/rgw/test_multi.md), and it isn't finding any of the test cases to run. i believe it's because the tests are no longer imported into this module
```
~/ceph/build $ cat test_multi.conf 
[DEFAULT]
num_zones = 2
num_ps_zones = 0
num_zonegroups = 1
gateways_per_zone = 1

~/ceph/build $ RGW_MULTI_TEST_CONF=test_multi.conf MON=1 OSD=1 MGR=0 MDS=0 nosetests ../src/test/rgw/test_multi.py

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
```